crates/evm/src/genesis.rs

```diff
@@ -25,6 +25,9 @@ pub struct AccountData {
     pub code_hash: B256,
     /// Smart contract code.
     pub code: Bytes,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// Smart contract storage
+    pub storage: Option<HashMap<U256, U256>>,
```

Comment:
No need for Option. This can be a hashmap with attrs: #[serde(default = "Default::default", skip_serializing_if = "HashMap::is_empty")]